settings/__init__.py

```diff
@@ -1,20 +1,16 @@
-import os
 import sys
 
-if os.getenv('TRAVIS', False):
-    from settings.travis import *  # noqa
-else:
+try:
+    from settings.local import *  # noqa
+except ImportError:
     try:
-        from settings.local import *  # noqa
+        from settings.base import *  # noqa
     except ImportError:
-        try:
-            from settings.base import *  # noqa
-        except ImportError:
-            sys.stderr.write(
-                "Error: Tried importing 'settings.local' and 'settings.base' "
-                "but neither could be found (or they're throwing an "
-                "ImportError). Please fix and try again.")
-            raise
+        sys.stderr.write(
+            "Error: Tried importing 'settings.local' and 'settings.base' "
+            "but neither could be found (or they're throwing an "
+            "ImportError). Please fix and try again.")
+        raise
```

Comment:
I was just thinking of doing that for exactly those reasons. I don't really see a need to support any of that any longer.